compiler/rustc_codegen_ssa/src/back/symbol_export.rs

```diff
@@ -577,6 +577,7 @@ pub(crate) fn linking_symbol_name_for_instance_in_crate<'tcx>(
     }
 
     let prefix = match &target.arch[..] {
+        "x86" if target.is_like_windows && undecorated.starts_with("?") => return undecorated,
```

Comment:
```suggestion
        "x86" | "x86_64" if target.is_like_msvc && undecorated.starts_with("?") => return undecorated,
```

This seems to more closely match what LLVM is doing

Reply:
I'm inclined to agree